azure-ad.sgnl.yaml

```diff
@@ -586,6 +634,16 @@ relationships:
     displayName: User Member
     fromAttribute: GroupMember.memberId
     toAttribute: User.id
+  RoleMember:
```

Comment:
Please reorganize this and put this above `UserMember`. Let's keep similar things grouped together and not intertwine relationships between different entities for readability.

And as requested in my previous comments, please add the role equivalents to the following relationships
```yaml
UserMember:
    name: Member
    displayName: User Member
    fromAttribute: GroupMember.memberId
    toAttribute: User.id
  GroupMember:
    name: Member
    displayName: Group Member
    fromAttribute: GroupMember.memberId
    toAttribute: Group.id
  MemberOf:
    name: MemberOf
    displayName: Member Of
    fromAttribute: GroupMember.groupId
    toAttribute: Group.id
  UserMemberGroup:
    name: Group
    displayName: User Member Group
    path:
      - relationship: UserMember
        direction: Backward
      - relationship: MemberOf
        direction: Forward
  GroupMemberGroup:
    name: Group
    displayName: Group Member Group
    path:
      - relationship: GroupMember
        direction: Backward
      - relationship: MemberOf
        direction: Forward
```

Reply:
Please check now